src/client/Atmosphere.js

```diff
@@ -96,6 +101,7 @@ export default class Atmosphere extends Environment {
           const errorObj = makeErrorObj(errors);
           reject(errorObj);
         } else {
+          // reject({_error: 'A really big one'})
```

Comment:
i was thinking about it, but i still use them anytime i need to slow down the response or see what an error looks like. still very helpful!